### PR TITLE
Test CleanUtil.cleanProjectFolder

### PR DESCRIPTION
**Pull request by mawHBT**
This is part of [peass-ci-issue #167](https://github.com/jenkinsci/peass-ci-plugin/issues/167)
"Provide an unit test, that calls CleanUtil.cleanProjectFolder (and checks that everything is completed, from a resource folder that is copied to target in the unit test)."

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -169,6 +169,13 @@
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-support</artifactId>
     </dependency>
+    <dependency>
+      <groupId>de.dagere.peass</groupId>
+      <artifactId>dependency</artifactId>
+      <version>0.2.3-SNAPSHOT</version>
```
Comment (DaGeRe) [inline pom.xml:175]: Please always use the latest and same Peass as the rest of the project (in this case: `${peass.version}`).

File: pom.xml [hunk 2]
```diff
@@ -189,6 +196,17 @@
           <excludeFilterFile>spotbugs-exclude.xml</excludeFilterFile>
         </configuration>
       </plugin>
+      <plugin>
```
Comment (DaGeRe) [inline pom.xml:199]: Is this really required? I thought this is only required when generating a test jar, not when using one.

Comment (mawHBT) [reply pom.xml]: I changed peass-version for dependency-module to ${peass.version} and removed the unnecessary maven-jar-plugin.

File: pom.xml (hunk 1 above)
Comment (mawHBT) [reply pom.xml]: I changed peass-version for dependency-module to ${peass.version}.

**Comment by DaGeRe** [comment]
Thanks for the PR, looks fine.

**Pull request merged**